### ISSUE TITLE
OpenGL.error.NullFunctionError: Attempt to call an undefined function glutInit, check for bool(glutInit) before calling

### ISSUE THREAD

**Issue by unholyeyebrows** [flathub/com.jetbrains.PyCharm-Community#81]
Hi

I previously raised the following issue which was fixed last year but the problem has returned:

https://github.com/flathub/com.jetbrains.PyCharm-Community/issues/60

Can this be fixed again? Thanks in advance?

Nick

**Comment by FakeShemp** [comment]
Nothing has changed as far as I know. Glut is still at the same location it used to be. Did PyOpenGL change its loading mechanics?

You should be able to copy the glut libraries to the same folder as your py script as a last resort.

**Comment by FakeShemp** [comment]
Solved in https://github.com/flathub/com.jetbrains.PyCharm-Community/commit/a666c37ec2f6cac81331ec13de73d6c0df611c9d

**Comment by unholyeyebrows** [comment]
Amazingly fast turn-around Rebecca. You rock!

Many thanks,
Nick

**Comment by avdodger** [comment]
it's August 2020 and this problem exists.  I've downloaded pyOpenGL and/or freeGlut to no avail.  I continue to get fails for using an undefined function glutInit()...  I've got .dlls in the directory where my code is, and other places in my path - they're all avoided.  I'm running either Python 3.8 using Idle or Anaconda with Spyder - nothing works.  Any ideas?  And yes, I'm only a couple months into Python.
